### ISSUE TITLE
Error outputting results after initial scans

### ISSUE THREAD

**Issue by genfemme** [bridgecrewio/checkov#5866]
When running multiple consecutive checkov scans in a GitHub Actions workflow, checkov fails to output results after the first 1-2 scans with the error:

```
Error: Unable to process file command 'env' successfully.

Error: Invalid format 'CHECKOV_RESULTS<<'. Name must not be empty and delimiter must not be empty
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'results<<'. Name must not be empty and delimiter must not be empty
```

The workflow steps taken:

1. Checkout code
2. Replace SSH with HTTPS 
3. Scan environment A (succeeds)
4. Scan environment B (succeeds) 
5. Scan environment C (fails with above error)

The action version used:
uses: bridgecrewio/checkov-action@master
OS: Ubuntu latest

Things tried:
- Setting output_file parameter

The error seems to indicate checkov is trying to save results to invalid environment variables after the initial scans.
Appreciate any help troubleshooting why checkov fails handling repeated scan result output!

**Comment by gruebel** [comment]
hey @genfemme thanks for reaching out.

The `checkov-action` saves the result output as an env var and it seems like when trying to run it multiple times it somehow overflows. You could try to reset it or just install the `checkov` CLI in your workflow and use as is.

**Comment by genfemme** [comment]
Hey @gruebel thanks for your reply.

I have added the step to Install Checkov via Pip in the Github workflow

I am getting the following error with the checkov cli:
  --download-external-modules: command not found 

I have tried the following options:
1)  --download-external-modules true
2)  --download-external-modules true quiet
Could you please advise if I'm missing something in my configuration or usage?

**Comment by genfemme** [comment]
After some trial and error, I found that structuring the full checkov command on a single line as a regular string resolved the "command not found" errors.

I was wondering if checkov supports scanning other file types besides the default of Terraform (.tf)? I have some scripts and YAML files I would like to scan as well.

I tried to get checkov to scan scripts and yaml files by using the absolute file paths and also pointing the -d directory flag to those locations, but neither approach worked to scan anything other than .tf files.

Is there a way to get checkov to analyze scripts and YAML configs? Or does it primarily support just Terraform right now? Let me know if you have any tips for getting it to work with other file extensions. Thanks!"
